cli/src/commands/extensions/extension.rs

```diff
@@ -55,6 +64,14 @@ impl Extension {
         &self.manifest.entry_point
     }
 
+    pub fn permissions(&self) -> &Permissions {
+        &self.manifest.permissions
+    }
+
+    pub fn requires_permissions(&self) -> bool {
+        !self.manifest.permissions.is_empty()
+    }
```

Comment:
Considering that `permissions` is already a public method, I'm not sure we're gaining much with this extra one.